app/policies/section_policy.rb

```diff
@@ -13,15 +13,31 @@ def initialize(user, section)
   #  - The template which they are modifying belongs to their org
   ##
 
-  def admin_create?
+  def index?
+    user.present?
+  end
+  
+  def show?
+    user.present?
```

Comment:
this policy is too open, isn't? I know they are not going to be able to modify anything but not sure if we want to allow anyone to access to the details of a section from a template that perhaps is under editing still...